Library/Homebrew/formulary.rb

```diff
@@ -263,6 +263,21 @@ def install
         raise "Cannot build from source from abstract formula."
       end
 
+      if (service_hash = json_formula["service"])
+        service_hash = Homebrew::Service.deserialize(service_hash)
+        run_params = service_hash.delete("run")
+        service do
+          if run_params.is_a?(Hash)
+            run(**run_params)
+          else
+            run run_params
+          end
```

Comment:
`Service#run` accepts either one param or kwargs so needs to be handled separately.

Reply:
How about keep_alive?

Reply:
https://github.com/Homebrew/brew/blob/8f6276ed5f1c7b8293128a481b175611cb2980c8/Library/Homebrew/service.rb#L121-L141

That accepts a string or a hash but always gets stored internally as a hash. It's not a problem if you just pass that hash to it then. It'd be trickier if it accepted kwargs.